src/components/Header/HeaderButton.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+
+import { Icon, Props as IconProps } from '../Icon'
+
+export interface HeaderButtonProps {
+  children?: React.ReactNode
+  url?: string
+  target?: string
+  icon?: IconProps['name']
+}
+
+const HeaderButtonComponent: React.FC<HeaderButtonProps & InjectedProps> = ({
+  theme,
+  ...props
+}) => (
+  <Wrapper theme={theme} href={props.url} target={props.target && props.target}>
```

Comment:
[MUST] If props.target is missing, 'undefined' will be entered in the target attribute